users/services/utils.py

```diff
@@ -42,3 +42,9 @@ def get_date_from_string(date_text):
     date_text = date_text.split('T')[0]
     date_text_datetime = datetime.strptime(date_text, DATETIME_FORMAT)
     return date_text_datetime.date()
+
+
+def sanitize_request_data(request_data):
+    for key, value in request_data.items():
```

Comment:
Thanks for implementing this important function.

Reply:
Sure. Without that function, our application may accept "    " as the company name, which will become a hassle in the future while the application is used.